packages/graph/package.json

```diff
@@ -7,12 +7,13 @@
     "directory": "packages/graph"
   },
   "license": "MIT",
-  "main": "dist/cjs/index.js",
-  "module": "dist/esm/index.js",
-  "types": "dist/type/index.d.ts",
+  "main": "dist/index.cjs",
+  "module": "dist/index.js",
```

Comment:
is this a dual package? `@rsdoctor/graph` runs on Node.js so the `module` field won't be recognized so it will always use the CJS entrypoint.